packages/sdk-components-react/src/time.tsx

```diff
@@ -327,6 +327,23 @@ const timeStyleOrUndefined = (
   return undefined;
 };
 
+export const parseDate = (datetimeString: string) => {
+  if (datetimeString === "") {
+    return;
+  }
+  const isNumber = /^\d+$/.test(datetimeString);
```

Comment:
01 is bad number
moreover it's check on positive Int can be done using JS i.e 
ordinary check isNumber +   Number.isInteger(value) && value >= 0;